### PR TITLE
[chore]: Remove old logger and cleanup logging

### PR DESCRIPTION
**Pull request by walmat**
## What changed (plus any additional context for devs)
Cleaned up the references to the old deprecated logger in `@utils/logger` and replaced the calls with `@/logger`. Also went ahead and added string references to the front of the logs to help us located the file that the log is coming from. This should help us debug quicker where an error is originating from. Along with all this work, I removed several instances in the app that called `logger.info` which sends a message to Sentry every time it's called and replaced it with `logger.debug` which is just added as a breadcrumb. This should help clean up events on Sentry that don't really have a purpose to be there.

### CODE REVIEW

File: src/design-system/docs/components/DocsAccordion/index.tsx [hunk 1]
```diff
@@ -58,7 +58,7 @@ export const DocsAccordion = ({ meta, description, examples }: Docs) => {
   );
 };
 
-////////////////////////////////////////////////////////////////////////////////
+// //////////////////////////////////////////////////////////////////////////////
```
Comment (jinchung) [inline src/design-system/docs/components/DocsAccordion/index.tsx:61]: seems like some autogenerated space here

File: src/design-system/docs/next-env.d.ts [hunk 2]
```diff
@@ -1,5 +1,5 @@
-/// <reference types="next" />
-/// <reference types="next/image-types/global" />
+// / <reference types="next" />
```
Comment (jinchung) [inline src/design-system/docs/next-env.d.ts:1]: seems like some autogenerated spaces here

File: src/handlers/ens.ts [hunk 3]
```diff
@@ -137,8 +135,9 @@ export const fetchMetadata = async ({
     const image_url = `https://metadata.ens.domains/mainnet/${contractAddress}/${tokenId}/image`;
     return { image_url, name };
   } catch (error) {
-    logger.sentry('ENS: Error getting ENS metadata', error);
-    captureException(new Error('ENS: Error getting ENS metadata'));
```
Comment (jinchung) [inline src/handlers/ens.ts:141]: looks like the `captureException` import no longer needed on line 4

File: src/logger/index.ts [hunk 4]
```diff
@@ -282,7 +282,7 @@ if (env.IS_DEV) {
   /**
    * Uncomment this to test Sentry in dev
    */
-  // logger.addTransport(sentryTransport);
+  logger.addTransport(sentryTransport);
```
Comment (jinchung) [inline src/logger/index.ts:285]: can we recommend this

File: src/resources/reservoir/mints.ts [hunk 5]
```diff
@@ -35,11 +35,11 @@ export const navigateToMintCollection = async (contractAddress: EthereumAddress,
         pricePerMint,
       });
     } else {
-      logger.warn('Mints: No collection found', { contractAddress, network });
+      logger.warn('[mints]: No collection found', { contractAddress, network });
       showAlert();
     }
   } catch (e) {
-    logger.warn('Mints: navigateToMintCollection error', {
+    logger.error(new RainbowError(`[mints]: navigateToMintCollection error`), {
```
Comment (jinchung) [inline src/resources/reservoir/mints.ts:42]: just want to confirm we want `error` and not `warn` here

File: src/model/wallet.ts [hunk 6]
```diff
@@ -1277,35 +1272,35 @@ export const loadSeedPhraseAndMigrateIfNeeded = async (id: RainbowWallet['id']):
     // First we need to check if that key already exists
     const keyFound = await keychain.hasKey(`${id}_${seedPhraseKey}`);
     if (!keyFound) {
-      logger.debug('[loadAndMigrate] - key not found, should need migration', {}, DebugContext.wallet);
+      logger.debug('[wallet]: key not found, should need migration');
```
Comment (jinchung) [inline src/model/wallet.ts:1275]: confirming if you no longer want the DebugContext.wallet attached

File: src/model/wallet.ts [hunk 7]
```diff
@@ -1277,35 +1272,35 @@ export const loadSeedPhraseAndMigrateIfNeeded = async (id: RainbowWallet['id']):
...
       // if it doesn't we might have a migration pending
       const isSeedPhraseMigrated = await keychain.loadString(oldSeedPhraseMigratedKey);
-      logger.debug(`[loadAndMigrate] - Migration pending? ${!isSeedPhraseMigrated}`, {}, DebugContext.wallet);
+      logger.debug(`[wallet]: Migration pending? ${!isSeedPhraseMigrated}`);
```
Comment (jinchung) [inline src/model/wallet.ts:1278]: same here: confirming if you no longer want the DebugContext.wallet attached

File: package.json [hunk 8]
```diff
@@ -42,8 +42,8 @@
     "format": "prettier --write .",
     "format:check": "prettier --check .",
     "lint": "yarn format:check && yarn lint:ts && yarn lint:js",
-    "lint:ci": "yarn format:check && yarn lint:ts && yarn lint:js --quiet",
-    "lint:js": "eslint --cache .",
+    "lint:ci": "yarn format:check && yarn lint:ts && yarn lint:js",
```
Comment (jinchung) [inline package.json:45]: making a note to make sure you want to keep these changes (not sure if it was while you were testing things)

**Review by jinchung — APPROVED** [review]
🌮  added a few comments just to make sure things were as intended - none are blockers

File: src/logger/index.ts (hunk 4 above)
Comment (walmat) [reply src/logger/index.ts:285]: oops good catch!

File: package.json (hunk 8 above)
Comment (walmat) [reply package.json:45]: Yeah so the --quiet flag previously wasn't doing anything. Attaching it on the js lint silences warnings (which I prefer and it speeds up linting by a significant amount since the console doesn't have to log 1000+ warnings).

TLDR: This was intentional
